aas-gui/Frontend/aas-web-gui/src/components/SubmodelPlugins/CarbonFootprint.vue

```diff
@@ -0,0 +1,180 @@
+<template>
+    <v-container fluid class="pa-0">
+        <!-- Header -->
+        <v-card class="mb-4">
+            <v-card-title>
+                <div class="text-subtitle-1">{{ "Carbon Footprint:" }}</div>
+            </v-card-title>
+        </v-card>
+        <!-- Product and Transport CarbonFootprint -->
+        <v-card class="mb-4 py-8">
+            <v-row flex align="center" justify="center">
+                <v-col cols="12" md="2" style="min-width: fit-content; padding-left: 20px; font-size: 20px;">
+                    <div v-if="productCarbonFootprintData">
+                        <p style="margin: 40px 0">
+                            <span
+                                style="display: inline-block; width: 14px; height: 14px; margin-right: 6px; border-radius: 3px"
+                                class="bg-pcf"></span>
+                            <span class="text-subtitle-1 subtitleText">Product:</span>
+                        <div class="text-h6">{{ productIsoValue + ' CO₂eq' }}</div>
```

Comment:
The Unit is missing here. It can be found in the ConceptDescription of the PCFCO2eq Property respectively the TCFCO2eq Property. The Unit can be fetched using the `SubmodelElementHandling` mixin like this:

```js
unitSuffix(PCFCO2eqProperty)
```